tephra-hbase-compat-1.1-base/src/main/java/org/apache/tephra/hbase/coprocessor/janitor/HBaseTransactionPruningPlugin.java

```diff
@@ -259,6 +262,8 @@ private long computePruneUpperBound(TimeRegions timeRegions) throws IOException
         if (pruneUpperBoundForTime != -1) {
           Long minPruneUpperBoundRegions = Collections.min(pruneUpperBoundRegions.values());
           return Math.min(pruneUpperBoundForTime, minPruneUpperBoundRegions);
+        } else {
+          LOG.debug("Ignoring invalid prune upper bound -1 for time {}", time);
```

Comment:
A better message would be: Ignoring regions for time {} because no pruneUpperBound was found for that time and the data must be incomplete.